specification/security/resource-manager/Microsoft.Security/preview/2024-07-01-preview/securityConnectors.json

```diff
@@ -0,0 +1,1707 @@
+          "description": "GCP project name"
+        }
+      }
+    },
+    "GcpOrganizationalData": {
+      "type": "object",
+      "description": "The gcpOrganization data",
+      "discriminator": "organizationMembershipType",
+      "required": [
+        "organizationMembershipType"
+      ],
+      "properties": {
+        "organizationMembershipType": {
+          "type": "string",
+          "description": "The multi cloud account's membership type in the organization",
+          "enum": [
+            "Member",
+            "Organization"
+          ],
+          "x-ms-enum": {
+            "name": "organizationMembershipType",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "GcpOrganizationalDataOrganization": {
+      "type": "object",
+      "description": "The gcpOrganization data for the parent account",
+      "x-ms-discriminator-value": "Organization",
+      "allOf": [
+        {
+          "$ref": "#/definitions/GcpOrganizationalData"
+        }
+      ],
+      "properties": {
+        "excludedProjectNumbers": {
+          "type": "array",
+          "description": "If the multi cloud account is of membership type organization, list of accounts excluded from offering",
+          "items": {
+            "type": "string",
+            "description": "account IDs"
+          }
+        },
+        "serviceAccountEmailAddress": {
+          "type": "string",
+          "description": "The service account email address which represents the organization level permissions container."
+        },
+        "workloadIdentityProviderId": {
+          "type": "string",
+          "description": "The GCP workload identity provider id which represents the permissions required to auto provision security connectors"
+        },
+        "organizationName": {
+          "readOnly": true,
+          "type": "string",
+          "description": "GCP organization name"
+        }
+      }
+    },
+    "GcpOrganizationalDataMember": {
+      "type": "object",
+      "description": "The gcpOrganization data for the member account",
+      "x-ms-discriminator-value": "Member",
+      "allOf": [
+        {
+          "$ref": "#/definitions/GcpOrganizationalData"
+        }
+      ],
+      "properties": {
+        "parentHierarchyId": {
+          "type": "string",
+          "description": "If the multi cloud account is not of membership type organization, this will be the ID of the project's parent"
+        },
+        "managementProjectNumber": {
+          "type": "string",
+          "description": "The GCP management project number from organizational onboarding"
+        }
+      }
+    },
+    "GithubScopeEnvironmentData": {
+      "type": "object",
+      "description": "The github scope connector's environment data",
+      "x-ms-discriminator-value": "GithubScope",
+      "allOf": [
+        {
+          "$ref": "#/definitions/EnvironmentData"
+        }
+      ],
+      "properties": {}
```

Comment:
what does this infer ? is this free flow on schema object for properties ? 
If not needed - please remove. 
If needed - please define the schema

Reply:
This version is build on top of the older version with the goal of supporting a new environment only, without the intention to make further unrelated changes